<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAgendarConsulta</class>
 <widget class="QDialog" name="DialogAgendarConsulta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agendar Consulta</string>
  </property>
  <widget class="QLabel" name="labelPet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pet:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPet">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="labelDono">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dono:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboDono">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="labelVet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Veterin√°rio:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboVeterinario">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
  </widget>

  <widget class="QLabel" name="labelData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>

  <widget class="QLabel" name="labelHora">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hora:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
  </widget>

  <widget class="QPushButton" name="btnConfirmar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmar Consulta</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
